datamodel/scripts/create-dumps.py

```diff
@@ -70,7 +74,7 @@ def create_plain_structure_only(database: str, version: str):
     print("::group::plain SQL structure only")
 
```

Comment:
PP: extensionSuffix verwenden.
```suggestion
    extensionSuffix = "" if len(extension_name) == 0 else f"_extenionSuffix"
```

Reply:
@cymed hier geht es nicht um die Benennung der Variable, sondern um den "_" vor dem Suffix. So bleibt man in naming konsistent und lesbar.
`tww_10.0_structure_agjahrgang.sql `ist lesbarerer als `tww_10.0_structureagjahrgang.sql.` Klar extremes Beispiel jedoch mit den Extensions denkbar.

Reply:
Underscore wird neu in argparse eingefügt, um **args zu nutzen